src/logger.cpp

```diff
@@ -1,158 +1,39 @@
-#include <memory>
-
-#include "KafkaW/ProducerTopic.h"
-#include "json.h"
 #include "logger.h"
-#include <atomic>
-#include <cstdarg>
-#include <cstdio>
-#include <cstdlib>
-#include <cstring>
-#include <memory>
 #include <string>
-#include <thread>
 #ifdef HAVE_GRAYLOG_LOGGER
 #include <graylog_logger/GraylogInterface.hpp>
 #include <graylog_logger/Log.hpp>
```

Comment:
I suspect the `graylog_logger` headers don't need to be included here anymore.

Reply:
Not including `graylog_logger` headers breaks the build. Despite using spdlog-graylog package you still need to explicitly include libraries both in cmake and here.